src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -54,7 +54,7 @@ private static Document mapScannedDocument(ScannedDocument doc) {
             return null;
         } else {
             return new Document(
-                doc.fileName,
+                getFileName(doc),
```

Comment:
here you are using 'doc.url.documentFilename' if it is not null

Reply:
Not exactly. I am using _doc.url.documentFilename_ if _doc.url_ is not null